.packit.yaml

```
@@ -29,7 +29,7 @@ jobs:
     trigger: pull_request
     targets:
       centos-stream-8:
-        additional_modules: "foreman:el8"
+        additional_modules: "foreman-devel:el8"
         additional_repos:
           - http://koji.katello.org/releases/yum/foreman-nightly/el8/x86_64/
```

can you please update this to 
```suggestion
          - http://yum.theforeman.org/releases/nightly/el8/x86_64/
```